install python extra scripts when repkging python

cyrush opened the pull request:
### Description

Resolves #17238

<!-- Please include a summary of the change and any other information and instructions to help reviewers understand the work -->

### Type of change
* [ ] Bug fix
* [ ] New feature
* [ ] Documentation update
* [x] Other <!-- please explain with a note below -->

CMake and install tweak

### How Has This Been Tested?
Tested on my mac, make sure the expected files landed in the install dir.

### Checklist:

<!-- For items in this checklist that do not apply, simply insert two tilde chars, `~~`, just ahead of the left bracket char, `[` at the beginning of a line. Each line ends with two tilde chars to make doing such ~~strikeouts~~ easy. -->

- [x] I have followed the [style guidelines][1] of this project.~~
- [x] I have performed a self-review of my own code.~~
- [x] I have commented my code where applicable.~~
~~- [] I have updated the release notes.~~
~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have added debugging support to my changes.~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works.~~
~~- [ ] I have confirmed new and existing unit tests pass locally with my changes.~~
~~- [ ] I have added new baselines for any new tests to the repo.~~
- [x] I have NOT made any changes to [*protocol* or *public interfaces*][3] in an RC branch.~~
- [x] I have assigned reviewers (see [VisIt's PR procedures][2] for more information).~~

[1]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/StyleGuide.html
[2]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers
[3]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/RCDevelopment.html#communication-protocols-and-public-apis

cyrush committed:
install python extra scripts when repkging python

cyrush committed:
make install change: make sure docs are built

brugger1 reviewed (dismissed):
Looks good. Do you want to add something to the release notes? I'll make a pass at cleaning up the release notes and can add something then if you don't have time.

cyrush committed:
udpate relnotes

cyrush commented:
@brugger1  I updated the relnotes.

I also added a fix that will build docs if you do a `make install` and didn't not explicitly build them before hand.

cyrush commented:
@brugger1  I am really confused b/c this small change doens't easily apply to develop.  I need to make sure we didn't miss changes on 3.2RC that should already be on develop.

cyrush commented:
applied to develop in #17255
